test/E2ETest/Microsoft.Test.E2E.AspNet.OData/DateTimeSupport/DateTimeTest.cs

```diff
@@ -83,6 +84,35 @@ public async Task ModelBuilderTest(string modelMode)
             Assert.False(modifiedDates.Type.IsNullable);
         }
 
+        [Theory]
+        [InlineData("convention")]
+        [InlineData("explicit")]
+        public async Task PostToDateTimeCollection(string modelMode)
+        {
+            await ResetDatasource();
+            //Arrange
```

Comment:
>//Arrange [](start = 12, length = 9)

If you want to add // Arrange, please add // Act // Assert as well.

Reply:
Adding Act & Assert as they are coupled tightly.